apps/condo/domains/common/components/TrackingContext.tsx

```diff
@@ -105,35 +110,48 @@ const useTracking: IUseTracking = () => {
         return fn as T
     }
 
-    const getEventName = (eventType: TrackingEventType) => {
-        const [domainName, isDetail, suffix] = compact(route.split('/'))
+    const getEventName: IGetEventName = (eventType) => {
+        const [domainName, detailPageName, suffix, customPage] = compact(route.split('/'))
         if (!domainName) {
             return ''
         }
-        const domain = domainName.charAt(0).toUpperCase() + domainName.slice(1)
+        const domain = upperFirst(domainName)
         let domainSuffix = 'Index'
         let domainPostfix = ''
 
-        switch (isDetail) {
+        switch (detailPageName) {
             case '[id]':
                 domainSuffix = 'Detail'
                 break
             case 'create':
-                domainSuffix = 'Create'
+            case 'update':
+            case 'hint':
+            case 'detail':
+            case 'signin':
+            case 'register':
+            case 'change-password':
+            case 'forgot':
```

Comment:
maybe put these values in an array and do something like this `if (array.includes(detailPageName)) { ... }`?

Reply:
Done